src/room/useGroupCall.ts

```diff
@@ -51,7 +51,7 @@ export interface UseGroupCallReturnType {
   requestingScreenshare: boolean;
   isScreensharing: boolean;
   screenshareFeeds: CallFeed[];
-  localDesktopCapturerSourceId: string;
+  localDesktopCapturerSourceId: string; // XXX: This looks unused?
```

Comment:
True, I guess there could be a use case but it doesn't seem like something to worry about now

Reply:
Oh yes - I commented this a while back when I was looking at strict mode failures but apparently never committed it... until now!